test/test_suite.py

```diff
@@ -937,6 +937,16 @@ def test_autoclose_on_insert(self):
         assert not r.returns_rows
 
 
+class CompoundSelectTest(_CompoundSelectTest):
+    @pytest.mark.skip("Spanner doesn't support ORDER BY clause in UNION subqueries")
+    def test_limit_offset_selectable_in_unions(self):
+        pass
+
+    @pytest.mark.skip("Spanner doesn't support ORDER BY clause in UNION subqueries")
```

Comment:
Please update these skip reasons to point to the previously mentioned bug if you still want to merge this PR.

I can confirm that both tests pass locally when the regex is fixed.

Reply:
Oh, you already fixes the regex?!
No, no, if there is a way to fix the error, I'll just close this PR.